esp/platform/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ include_directories (
          ./../../system/jlib 
          ./../bindings/http/platform 
          ./../../common/environment 
+         ./../../common/workunit
```

Comment:
I would have expected a need to add workunit to the link libraries as well.

Reply:
Not sure why it builds without - probably because one of the libraries included already links to it. But I will add.